app/invocation/invocation_not_found.tsx

```diff
@@ -14,20 +17,12 @@ export default class InvocationNotFoundComponent extends React.Component {
   }
 
   render() {
-    return (
-      <div className={this.props.authorized ? "state-page" : "login-interstitial"}>
-        {this.props.authorized && (
-          <div className="shelf">
-            <div className="container">
-              <div className="breadcrumbs">Invocation {this.props.invocationId}</div>
-              <div className="titles">
-                <div className="title">Invocation not found!</div>
-              </div>
-              <div className="details">Double check your invocation URL and try again.</div>
-            </div>
-          </div>
-        )}
-        {!this.props.authorized && (
+    const invocationExists = this.props.error?.code !== "NotFound";
+    const canLogin = capabilities.auth && !this.props.user;
+
+    if (invocationExists && canLogin) {
```

Comment:
this looks reasonable, just make sure to test it in
- open source
- enterprise with no auth configured
- enterprise with auth configured, logged out
- enterprise with auth configured, logged in